cmd/operator/main.go

```diff
@@ -132,7 +141,39 @@ func runCLI(logger *logrus.Logger, cfg *config.Config) {
 			logger.Info("Rollout process succeeded")
 		}
 
-		interval := time.Duration(cfg.Rollout.Interval)
+		interval := time.Duration(serviceConfig.Strategy.Interval)
 		time.Sleep(interval * time.Second)
 	}
 }
+
+func flagsAreValid() (bool, error) {
```

Comment:
might wanna also check flRegions doesn't contain `""` empty strings e.g. `-flag=,,,,`

Reply:
I was assuming that if flRegions is empty, the operator queries all the available regions.

Reply:
yeah but what if `-regions=a,,,b`

Reply:
Oh, I got you. In theory, we could also check if the regions are valid too. Is that something worth doing?

In the follow-up PR, I would add a list of available regions, so we can use that one if `regions` is empty.

Reply:
> Is that something worth doing?
not needed, API already validates that (probably with a 404 or a dns lookup error saying nonexistingregion-run.googleapis.com not found)